R/nbImport.R

```diff
@@ -8,11 +8,13 @@
 #' @param ploidy average copy number in the tumor sample.
 #' @param sig.assign Logical. If TRUE, each variant will be assigned to a
 #' mutational signature.
-#' @param assign.method Method to assign signatures: "max" to assign the
-#' signature with the highest probability, "sample" to randomly assign based on
+#' @param assign.method Method to assign signatures: `max` to assign the
+#' signature with the highest probability, `sample` to randomly assign based on
 #' signature probabilities.
 #' @param ID sample name.
-#' @param sig.file File path to the SigAssignment output file, typically named
+#' @param sig.file Either the output of `fit_to_signatures` from package
```

Comment:
Please correct me if I'm wrong. `sig.file` is always assumed to be a text file, but the `fit_to_signature` output is an R object. I think this causes error because we are running `fread` in `.assign_signatures` line 175.

This should be properlz clarified